### PR TITLE
Replace unicode column control headers with FontAwesome

### LINKED ISSUE #3918: Data Viewer pager's icons don't render correctly on macOS
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1135
    OS Version      : macOS 10.14.1
    R Version       : 3.5.1

### Steps to reproduce the problem

Open the data viewer with a 'wide' dataset, e.g.

``` r
View(matrix(1, ncol = 100, nrow = 10))
```

### Describe the problem in detail

The icons for the pager show as white boxes:

screen shot 2018-11-16 at 9 34 05 am

### Describe the behavior you expected

Nicely-rendered arrows / icons.

### PR DESCRIPTION
**Pull request by adamconroy**
Push to fix #3918 

I'm opening this as a PR because I'm adding the FontAwesome library to the base index.html file of RStudio. I've trimmed it to the bare minimum library but it still adds 1.5 megs to the load. This seems to be something that is not yet established so I wanted to make sure that this isn't putting anything in danger. The big plus side is we could then lean on FA icons for overhauling our UX to make it look more modern moving forward without hiring a graphic designer.

se5wqff

### CODE REVIEW

File: src/cpp/session/resources/grid/datatables/css/dataTables.scroller.css [hunk 1]
```diff
@@ -36,9 +36,4 @@ div.DTS div.dataTables_scrollFoot {
 
 div.DTS div.dataTables_scrollBody {
 	z-index: 2;
-}
-
-div.DTS div.dataTables_scroll {
-	background: url('../images/loading-background.png') repeat 0 0;
-}
```
Comment (adamconroy) [inline src/cpp/session/resources/grid/datatables/css/dataTables.scroller.css]: This file doesn't exist, it was throwing an error every time the table was loading.

File: src/gwt/src/org/rstudio/core/client/theme/res/themeStyles.css [hunk 2]
```diff
@@ -2469,7 +2469,7 @@ body.ubuntu_mono .searchBox {
 .dataTableColumnWidget {
    font-size: 10px;
    border: none;
-   width: 50px;
+   width: 60px;
```
Comment (adamconroy) [inline src/gwt/src/org/rstudio/core/client/theme/res/themeStyles.css]: Looks nice for up to 10,000 columns at this width.

**Comment by jjallaire** [comment]
There are ways to add individual font awesome icons rather than taking the whole library. @rich-iannone knows how to do this.

**Comment by adamconroy** [comment]
@jjallaire thanks JJ, I looked into it and figured out how to make a custom icon font pack. Went from 2 megs to 3k added. I'm closing this PR and opening https://github.com/rstudio/rstudio/pull/3929 so the big binary file doesn't get added into the git history.

**Pull request closed without merging**